docker-compose.yml

```diff
@@ -27,7 +27,7 @@ services:
         build:
             context: geth
         volumes:
-            - geth:/root/.ethereum
+            - ./geth-data:/root/.ethereum
```

Comment:
I think we should probably make it obvious that its here and not have it as a hidden directory

Reply:
@AgeManning perhaps you misread the path, and placement of the dot? `./geth-data` should not be a hidden directory

Reply:
Like @hukkinj1 is saying. We are "mounting" local dir `geth-data` to the `/root/.ethereum` in the container. It is the default place of Geth datafiles. The same way `/root/.lighthouse` are. I am not a big fan of the hidden dirs there either, but that is how it is :D